lib/board.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :sq_val
+
+  def initialize
+    @sq_val = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  def draw_board
+    puts '  ¹     | ²     | ³     '
+    puts '        |       |       '
+    puts "    #{@sq_val[0]}   |   #{@sq_val[1]}   |   #{@sq_val[2]}   "
+    puts '        |       |       '
+    puts '        |       |       '
+    puts ' -----------------------'
+    puts '  ⁴     | ⁵     | ⁶     '
+    puts '        |       |       '
+    puts "    #{@sq_val[3]}   |   #{@sq_val[4]}   |   #{@sq_val[5]}   "
+    puts '        |       |       '
+    puts '        |       |       '
+    puts ' -----------------------'
+    puts '  ⁷     | ⁸     | ⁹     '
+    puts '        |       |       '
+    puts "    #{@sq_val[6]}   |   #{@sq_val[7]}   |   #{@sq_val[8]}   "
+    puts '        |       |       '
+    puts '        |       |       '
+  end
```

Comment:
All display codes should be in the main.rb file, this is the entry and output point of your program.

Reply:
We were doing it before and the previous TSE told us we should create a class for Board.rb. The big question is, why to have a class Board if we have to draw the board on main.rb?